SixTrack/sixtrack.s

```diff
@@ -56185,17 +56316,29 @@ c$$$            endif
      &g16.10/14x,a16,2x,g16.10,1x,g16.10/14x,a16,2x,g16.10,1x,g16.10)
       end
 +dk postpr
++if .not.stf
 +if .not.cr
       subroutine postpr(nfile)
 +ei
 +if cr
       subroutine postpr(nfile,nnuml)
 +ei
++ei
++if stf
++if .not.cr
+      subroutine postpr(posi)
++ei
++if cr
+      subroutine postpr(posi,nnuml)
++ei
++ei
++ei
```

Comment:
Seems you have one extra +ei here? Check that you're not missing a +if somewhere...